cmd/tailscaled/tailscaled.go

```diff
@@ -96,6 +97,34 @@ var subCommands = map[string]*func([]string) error{
 	"debug":                   &debugModeFunc,
 }
 
+func tryWindowsAppDataMigration() {
+	if runtime.GOOS != "windows" {
+		return
+	}
+
+	localAppData := filepath.Join(os.Getenv("LocalAppData"), "Tailscale")
```

Comment:
maybe call this variable `oldDir` for clarity? both that it's the old location and that it's a directory and not a file.

Reply:
Done.